components/engine/integration-cli/docker_cli_build_test.go

```diff
@@ -6234,6 +6234,28 @@ func (s *DockerSuite) TestBuildCopyFromResetScratch(c *check.C) {
 	c.Assert(strings.TrimSpace(res), checker.Equals, "")
 }
 
+func (s *DockerSuite) TestBuildMultiStageParentConfig(c *check.C) {
```

Comment:
Should we add this test to the https://github.com/moby/moby repository as well?

Reply:
hopefully with the refactor we should be able to cover this with a unit test in moby/moby